my_store/blog/dashboard/views.py

```diff
@@ -0,0 +1,64 @@
+# from django.shortcuts import render
```

Comment:
See how each module should order by this document https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/